### DISCUSSION TITLE
Welcome to OpenResearch Discussions!

### DISCUSSION THREAD

**Discussion by WolfgangFahl** [SmartDataAnalytics/OpenResearch#33] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MusaabKh** [comment]


**Reply by WolfgangFahl** [reply]
We just recently changed the Sidebar of open research to reflect the decisison to focus on Event and EventSeries see 
https://www.openresearch.org/wiki/Development_guidelines and https://www.openresearch.org/mediawiki/index.php?title=MediaWiki%3ASidebar&type=revision&diff=228545&oldid=228168

**Reply by WolfgangFahl** [reply]
Could you please make this a discussion topic - it is a valid and important question.

**Reply by WolfgangFahl** [reply]
https://github.com/SmartDataAnalytics/OpenResearch/discussions/new

**Reply by WolfgangFahl** [reply]
Please also mention the wikipush query you were using.
